### PR TITLE
Upgrade source plugin and adjust breaking changes

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Since the new version of the source plugin have been done - upgrading.

### CODE REVIEW

File: recipe-client.log [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[started client]:
```
Comment (makma) [inline recipe-client.log:1]: .gitignore?

File: recipe-server.log [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (makma) [inline recipe-server.log:1]: .gitignore?

File: recipe-client.log (hunk 1 above)
Comment (Simply007) [reply recipe-client.log:1]: Will ignore

File: recipe-server.log (hunk 2 above)
Comment (Simply007) [reply recipe-server.log:1]: Will ignore

**Pull request merged**